pkg/build/api/v1/types.go

```diff
@@ -303,6 +303,10 @@ type BuildStrategy struct {
 
 	// CustomStrategy holds the parameters to the Custom build strategy
 	CustomStrategy *CustomBuildStrategy `json:"customStrategy,omitempty"`
+
+	// JenkinsPipelineStrategy holds the parameters to the Jenkins Pipeline build strategy.
+	// This strategy is experimental.
```

Comment:
We can even live with that.

On Wed, Apr 27, 2016 at 6:08 PM, Jessica Forrester <notifications@github.com
> wrote:

> In pkg/build/api/v1/types.go
> <https://github.com/openshift/origin/pull/7949#discussion_r61345321>:
>
> > @@ -303,6 +303,10 @@ type BuildStrategy struct {
> >
> >  	// CustomStrategy holds the parameters to the Custom build strategy
> >  	CustomStrategy *CustomBuildStrategy `json:"customStrategy,omitempty"`
> > +
> > +	// JenkinsPipelineStrategy holds the parameters to the Jenkins Pipeline build strategy.
> > +	// This strategy is experimental.
>
> Showing what exactly? Cause @spadgett <https://github.com/spadgett> 's
> changes to tolerate these build strategies merged with the 1.3 branch a
> little while ago, it shows / lets you edit the Jenkinsfile
> … <#m_9053557025395677933_>
> On Wed, Apr 27, 2016 at 5:41 PM, Clayton Coleman ***@***.***> wrote: In
> pkg/build/api/v1/types.go <#7949 (comment)
> <https://github.com/openshift/origin/pull/7949#discussion_r61341608>>: >
> @@ -303,6 +303,10 @@ type BuildStrategy struct { > > // CustomStrategy
> holds the parameters to the Custom build strategy > CustomStrategy
> *CustomBuildStrategy `json:"customStrategy,omitempty"` > + > + //
> JenkinsPipelineStrategy holds the parameters to the Jenkins Pipeline build
> strategy. > + // This strategy is experimental. What's the timing on the
> image? As long as the UI isn't showing it yet i'm not worried — You are
> receiving this because you were mentioned. Reply to this email directly or
> view it on GitHub <
> https://github.com/openshift/origin/pull/7949/files/fb7993e7a126f0d0a611f5d9aa5fad48b35d5e76#r61341608>
>
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/7949/files/fb7993e7a126f0d0a611f5d9aa5fad48b35d5e76#r61345321>
>